content/news/buzz/2025-curtin-capstone-semester-end-blog/contents.lr

```diff
@@ -0,0 +1,102 @@
+title: 2025 Curtin University Capstone Project
+---
+author: Curtin Capstone Team
+---
+body:
+
+Over the past semester, a team of final year students from [Curtin University](https://www.curtin.edu.au) has been collaborating with the BeeWare Project as part of a capstone project for their degrees. Here's a summary of what they've done so far this semester, and what they have planned for the rest of the year.
+
+## Curtin Capstone
+
+Capstone is the final year project undertaken by students across all computing disciplines at Curtin University, including Computer Science, Software Engineering, Cyber Security, and Information Technology. It allows students the opportunity to work in teams on real-world projects in collaboration with industry partners, gaining practical experience and professional exposure before graduating.
+
+This year our team has had the exciting opportunity to contribute to BeeWare for our Capstone project.
+
+Meet the Team:
+
+ - Kavidu Abeykoon Mudiyanselagedara
+	 - Discipline: Information Technology
+	 - GitHub: [kavi2du](https://github.com/kavi2du)
```

Comment:
I think we can tighten this up a bit; suggested format:
```suggestion
 - Kavidu Abeykoon Mudiyanselagedara ([kavi2du](https://github.com/kavi2du)); Information Technology
```
rinse and repeat for the other bullet points.